deepspeed/module_inject/replace_module.py

```diff
@@ -468,23 +429,24 @@ def replace_with_policy(child,
                     rotary_dim=rotary_dim,
                     mlp_after_attn=(rotary_dim is None or rotary_dim < 0),
                     mlp_act_func_type=policy.mlp_act_func_type,
-                    training_mp_size=training_mp_size,
+                    training_mp_size=config.training_mp_size,
                     bigscience_bloom=bigscience_bloom,
-                    max_out_tokens=max_out_tokens,
+                    max_out_tokens=config.max_out_tokens,
                     scale_attn_by_inverse_layer_idx=scale_attn_by_inverse_layer_idx)
 
-            if quantize and quantize_settings is not None:
+            if quantize and config.quantize_settings is not None:
```

Comment:
Is `config.quantize_settings` a parameter? I think you want `config.quant`. Also, the default value for `config.quant` is `{}` and this is not equivalent to `None`

Reply:
oh, good catch. quantize_settings is a local thing and that has to be cleaned up in a separate PR after Reza's int4 PR gets merged. 

Let me fix this part differently for now.

Reply:
Removing quantize_settings completely as we don't use that code path anymore based on Reza's feedback.